Customization/T250/CodeSnippets/Step1.5.1/RS401000.aspx

```diff
@@ -45,7 +45,7 @@
           <%-- The added code --%>
           <RowTemplate>
             <px:PXSelector ID="edInvoiceNbr" runat="server"
-                DataField="InvoiceNbr" Enabled="False" AllowEdit="True" />
```

Comment:
It is not needed if the field is disabled in code. Developers should avoid field disabling in aspx, because the edit-ability of the field is not the UI's responsibility, it's the responsibility of business logic.